tool/tsh/tsh.go

```diff
@@ -629,6 +631,7 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 	aws.Flag("app", "Optional Name of the AWS application to use if logged into multiple.").StringVar(&cf.AppName)
 	aws.Flag("endpoint-url", "Run local proxy to serve as an AWS endpoint URL. If not specified, local proxy serves as an HTTPS proxy.").
 		Short('e').Hidden().BoolVar(&cf.AWSEndpointURLMode)
+	aws.Flag("aws-command", "Execute different commands (e.g. terraform) under Teleport credentials").StringVar(&cf.AWSCommand)
```

Comment:
I think the flag name should be `--` so it's consistent with how other CLI tools accept extra command arguments.

Reply:
Personally, I would keep `tsh aws` as it is. Here are my reasonings:
- `tsh aws/az/gcloud` are designed to be simple replacements for `aws/az/gcloud` command.
- I would call it a "coincident" that the setup for AWS CLI works for `terraform`. Many applications require extra setup (e.g.  `NODE_EXTRA_CA_CERTS=<ca-path>` for node, `AWS_SDK_LOAD_CONFIG=true` for golang SDK v2, trust store for Java)

I would propose to add this functionality to the proxy command, e.g.:
```
tsh proxy aws -- terraform init
```
The proxy command is more flexible with `-p` or `-f` flat etc. And possibly we can do cool things like these in the future.
```
tsh proxy aws --format nodejs -- my-node-program
tsh proxy aws --app aws-account-1 --app aws-acount-2 -- terraform init
```

If we agree on `tsh proxy aws -- <command and args>`, I can make a draft PR for that.

If we prefer to keep it in `tsh aws`, I personally like `--exec` better than `--aws-command`. Though I may prefer `--exec` to be a boolean. I don't think we can use `--` as a flag name directly. `--` is built-in with special logic for separating flags from positional arguments.

Reply:
> Can we rename this to `--exec` flag. I think that duplicate the aws prefix like `tsh aws app1 --aws-comand ...` so the UI will look as follow: `tsh aws app1 --exec ...`
> 
> Also we have `tsh az` `tsh gcp` command and the `--exec` flag seems to be more generic. @greedy52 wdyt ?

Yeah, I can push this change, I think exec is a lot better than aws-command as a flag, even not considering azure and gcp functionality you guys have.